src/_shared/js/ads.js

```diff
@@ -11,17 +11,25 @@ const GLABS_EDITION = {
     US: 'guardian-labs-us'
 };
 
-function addTrackingPixel(rootNode) {
+function addTrackingPixel() {
     if( !trackingPixel ) return;
+    loadImage(trackingPixel);
+}
+
+function addPixel(url) {
+    loadImage(url);
+}
 
-    if( !pixelTemplate ) {
-        pixelTemplate = document.createElement('img');
-        pixelTemplate.className = 'creative__pixel';
-    }
+function addViewabilityTracker(creativeId) {
+    var tracker = document.getElementById('viewabilityTracker');
+    if( !tracker ) return;
+    tracker = tracker.firstChild.nodeValue.trim();
```

Comment:
Hmm.... 🤔 Cool, if we are in 'control' of the DOM tree then no worries about missing children 😅 👍 

At the moment I don't really know the purpose of UNIQUE_ID. What do we use the values for later? 

IIUC: `fabric-video-expandable-` will end up with a random number appended... What then reads these? Do they end up in Ophan / GA?